pkg/assets/crd.go

```diff
@@ -159,7 +159,7 @@ func ApplyCRDs(cfg *config.MicroshiftConfig) error {
 		}
 		c := readCRDOrDie(crdBytes)
 		if err := wait.Poll(customResourceReadyInterval, customResourceReadyTimeout, func() (bool, error) {
-			if err := applyCRD(client, c); err != nil {
+			if err := applyCRD(client, c, ctx); err != nil {
```

Comment:
What do you think about changing the order of so that `ctx` is the first parameter to keep things stylistically the same with other uses?

Reply:
Ah, yes, definitely. Not sure why I decided to switch it up :D good catch!